examples/README.md

```diff
@@ -7,3 +7,23 @@ In this folder there are examples taken from the delta.io quickstart guide and d
 ### Instructions
 * To run an example in Python run `spark-submit --packages io.delta:delta-core_2.12:{Delta Lake version} PATH/TO/EXAMPLE`
 * To run the Scala examples, `cd examples/scala` and run `./build/sbt "runMain example.{Example class name}"` e.g. `./build/sbt "runMain example.Quickstart"`
+
+### Docker Instructions
+
+#### Building the image
+
+* To build the image run: `docker build -t delta-lake-playground:latest .`
+
+#### Running the image
+
+* After the image is built run jupyterlab using: `docker run -it -p 8888:8888 delta-lake-playground`
+* Open Jupyter in your browser `http://localhost:8888/lab?token=docker`
+
+#### Sample Notebooks
+* There are sample python notebooks which are ported in the notebooks folder.
```

Comment:
have you test them?

Reply:
also .. elaborate on how to run both scala and python notebooks. give precise instructions.